sdk/nodejs/api/utils.ts

```diff
@@ -7,55 +7,62 @@ import {
 } from "../common/errors/index.js"
 import { QueryTree } from "./client.gen.js"
 
-function buildArgs(item: any): string {
-  const entries = Object.entries(item.args)
-    .filter((value) => value[1] !== undefined)
-    .map((value) => {
-      return `${value[0]}: ${JSON.stringify(value[1]).replace(
-        /\{"[a-zA-Z]+"/gi,
-        (str) => str.replace(/"/g, "")
-      )}`
-    })
-  if (entries.length === 0) {
+/**
+ * Format argument into GraphQL query format.
+ */
+function buildArgs(args: any): string {
+  // Remove unwanted quotes
+  const formatValue = (v: string) =>
```

Comment:
nit: could you use `value` instead of `v` to actually improve code readability.

Reply:
Done in https://github.com/dagger/dagger/pull/4274/commits/2037109cf407be1b4055baa9508a66ff58fb940b